crates/astria-sequencer/src/faucet/action.rs

```diff
@@ -0,0 +1,41 @@
+use crate::account::types::Balance;
+use astria_proto::{
+    native::sequencer::v1alpha1::Address,
+};
+
+/// Represents a request for funds from the faucet.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Request {
```

Comment:
Early feedback: after the (un)signed transactions are moved out of astria-sequencer it must no longer export any types except for its config and the `Sequencer` type as an entrypoint to the service.

Reply:
yeah agreed, however all the different actions for each module needs to be moved to a different crate as well, would rather not do that here. if we know what crate we're moving them to, i can make a PR to move them out